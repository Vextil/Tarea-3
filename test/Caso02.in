# Binario
# hoja
CrearBinario solo 5 1 FIN
AlturaCantidadBinario 1
Izquierdo 1 2
Derecho 1 3
Linealizar 1 1
ImprimirBinario 1
ImprimirLista 1
BuscarABB 1 no_esta
FiltroBinario 1 2 < 4
FiltroBinario 1 2 > 5
ImprimirBinario 4
# raiz e hijo izquierdo
CrearBinario tiene 5 1 izquierdo 1 FIN
AlturaCantidadBinario 1
Izquierdo 1 2
Derecho 1 3
ImprimirBinario 2
Linealizar 1 1
ImprimirBinario 1
ImprimirLista 1
BuscarABB 1 no_esta
FiltroBinario 1 2 < 4
FiltroBinario 1 2 > 5
ImprimirBinario 4
ImprimirBinario 5
# raiz e hijo derecho
CrearBinario con 5 1 derecho 1 FIN
AlturaCantidadBinario 1
Izquierdo 1 2
Derecho 1 3
ImprimirBinario 3
Linealizar 1 1
ImprimirBinario 1
ImprimirLista 1
BuscarABB 1 no_esta
FiltroBinario 1 2 < 4
FiltroBinario 1 2 > 5
ImprimirBinario 4
ImprimirBinario 5
# Balanceado de 1 y 2 elementos
CrearLista 1 solo FIN
CrearBalanceado 1 1
AlturaCantidadBinario 1
CrearLista 1 con derecho FIN
CrearBalanceado 1 1
ImprimirBinario 1
# arbol hacia la derecha
CrearBinario ahora 0 1 beber 1 con 2 feliz 3 jugo 4 loco 5 otro 6 que 7 rie 8 sin 9 tomar 10 whisky 11 FIN
ImprimirBinario 1
Linealizar 1 1
ImprimirLista 1
# arbol hacia la izquierda
CrearBinario whisky 0 1 tomar 1 sin 2 rie 3 que 4 otro 5 loco 6 jugo 7 feliz 8 con 9 beber 10 ahora 11 FIN
ImprimirBinario 1
Linealizar 1 1
ImprimirLista 1
# balanceado de 12
CrearLista 1 ahora beber con feliz jugo loco otro que rie sin tomar whisky FIN
CrearBalanceado 1 1
AlturaCantidadBinario 1
ImprimirBinario 1
# otros
CrearBinario martes 5 1 dia 4 gran 4 altura 2 tambien 9 zoo 12 water 43 ya 12 FIN
AlturaCantidadBinario 1
FiltroBinario 1 2 < 4
BuscarABB 1 ahora
BuscarABB 1 dia
Linealizar 1 2
Izquierdo 1 2
Derecho 1 3
RemoverBinario 1 ahora
RemoverBinario 1 tambien
ImprimirBinario 1
ImprimirBinario 2
ImprimirBinario 3
ImprimirBinario 4
ImprimirLista 2
Fin
