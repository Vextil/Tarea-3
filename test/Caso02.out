1># Binario
2># hoja
3>OK
4>Altura: 1; Cantidad: 1
5>No tiene hijo izquierdo.
6>No tiene hijo derecho.
7>OK
8>
 (5,solo)
9>Ordenada. solo 
10>No se encontro el texto.
11>OK
12>El filtro devolvio un arbol sin elementos.
13>
 (5,solo)
14># raiz e hijo izquierdo
15>OK
16>Altura: 2; Cantidad: 2
17>OK
18>No tiene hijo derecho.
19>
 (1,izquierdo)
20>OK
21>
 (5,tiene)
  (1,izquierdo)
22>Ordenada. izquierdo tiene 
23>No se encontro el texto.
24>OK
25>OK
26>
 (5,tiene)
27>
 (1,izquierdo)
28># raiz e hijo derecho
29>OK
30>Altura: 2; Cantidad: 2
31>No tiene hijo izquierdo.
32>OK
33>
 (1,derecho)
34>OK
35>
  (1,derecho)
 (5,con)
36>Ordenada. con derecho 
37>No se encontro el texto.
38>OK
39>OK
40>
 (5,con)
41>
 (1,derecho)
42># Balanceado de 1 y 2 elementos
43>OK
44>OK
45>Altura: 1; Cantidad: 1
46>OK
47>OK
48>
 (0,derecho)
  (0,con)
49># arbol hacia la derecha
50>OK
51>
            (11,whisky)
           (10,tomar)
          (9,sin)
         (8,rie)
        (7,que)
       (6,otro)
      (5,loco)
     (4,jugo)
    (3,feliz)
   (2,con)
  (1,beber)
 (0,ahora)
52>OK
53>Ordenada. ahora beber con feliz jugo loco otro que rie sin tomar whisky 
54># arbol hacia la izquierda
55>OK
56>
 (0,whisky)
  (1,tomar)
   (2,sin)
    (3,rie)
     (4,que)
      (5,otro)
       (6,loco)
        (7,jugo)
         (8,feliz)
          (9,con)
           (10,beber)
            (11,ahora)
57>OK
58>Ordenada. ahora beber con feliz jugo loco otro que rie sin tomar whisky 
59># balanceado de 12
60>OK
61>OK
62>Altura: 4; Cantidad: 12
63>
   (0,whisky)
    (0,tomar)
  (0,sin)
   (0,rie)
    (0,que)
 (0,otro)
   (0,loco)
    (0,jugo)
  (0,feliz)
    (0,con)
   (0,beber)
    (0,ahora)
64># otros
65>OK
66>Altura: 5; Cantidad: 8
67>OK
68>No se encontro el texto.
69>
  (4,gran)
 (4,dia)
  (2,altura)
70>OK
71>OK
72>OK
73>OK
74>OK
75>
  (12,zoo)
    (12,ya)
   (43,water)
 (5,martes)
   (4,gran)
  (4,dia)
   (2,altura)
76>
  (4,gran)
 (4,dia)
  (2,altura)
77>
  (12,zoo)
    (12,ya)
   (43,water)
 (9,tambien)
78>
   (12,zoo)
     (12,ya)
    (43,water)
  (9,tambien)
 (5,martes)
   (4,gran)
  (4,dia)
79>Ordenada. altura dia gran martes tambien water ya zoo 
80>Fin
